libraries/concurrent-merkle-tree/src/state.rs

```diff
@@ -0,0 +1,102 @@
+use crate::utils::hash_to_parent;
+
+/// Stores the path of nodes changed in a tree by a Merkle tree operation
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+#[repr(C)]
+pub struct ChangeLog<const MAX_DEPTH: usize> {
```

Comment:
Same point here on the `Pod` / `Zeroable`, is it possible?

Reply:
Impossible for const generics

Reply:
Ah what a shame, it looks like a generic that gives a size is treated the same as any other generic